back-end/department/models.py

```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from basemodel.models import BaseModel
+from django.contrib.auth.models import User
+
+
+# Create your models here
+
+class Department(BaseModel):
+    name = models.TextField(blank=True,null=True,max_length=50)
+    shortcut = models.TextField(blank=True,null=True,max_length=10)
+    description = models.TextField(blank=True,null=True,max_length=5000)
```

Comment:
Any specific reason for requiring this?

Reply:
We can use enum but this helps us to store HoD and DFA details if required. This also helps to store department specific details in description or urls if any which could be displayed on website. Similiar details are present in Placement broucher

Reply:
Shall I resolve converation